src/functions/module_statistical.ts

```diff
@@ -555,6 +803,105 @@ export const COVARIANCE_S = {
   isExported: true,
 } satisfies AddFunctionDescription;
 
+// -----------------------------------------------------------------------------
+// FORECAST
+// -----------------------------------------------------------------------------
+export const FORECAST: AddFunctionDescription = {
+  description:
+    "Calculates the expected y-value for a specified x based on a linear regression of a dataset.",
+  args: [
+    arg("x (number)", _t("The value on the x-axis to forecast.")),
+    arg(
+      "data_y (range<number>)",
+      _t("The range representing the array or matrix of dependent data.")
+    ),
+    arg(
+      "data_x (range<number>)",
+      _t("The range representing the array or matrix of independent data.")
+    ),
+  ],
+  returns: ["NUMBER"],
+  compute: function (
+    x: Maybe<CellValue>,
+    dataY: Matrix<CellValue>,
+    dataX: Matrix<CellValue>
+  ): number {
+    const { flatDataX, flatDataY } = filterAndFlatData(dataY, dataX);
+    return predictLinearValues(
+      [flatDataY],
+      [flatDataX],
+      toMatrix(toNumber(x, this.locale)),
+      true
+    )[0][0];
+  },
+  isExported: true,
+};
+
+// -----------------------------------------------------------------------------
+// GROWTH
+// -----------------------------------------------------------------------------
+export const GROWTH: AddFunctionDescription = {
+  description: "Fits points to exponential growth trend.",
+  args: [
+    arg(
+      "known_data_y (range<number>)",
+      _t(
+        "The array or range containing dependent (y) values that are already known, used to curve fit an ideal exponential growth curve."
+      )
+    ),
+    arg(
+      "known_data_x (range<number>, default=[1,2,3,...])",
```

Comment:
![image](https://github.com/odoo/o-spreadsheet/assets/53175338/5e5205c0-c766-4fee-844b-954c7555cce0)

Default value not supported. 
The problem is the comma.

we must either change ```makeArg``` to take into account commas at this level or not use the comma but something else

Reply:
@LucasLefevre what do you think ?

Reply:
use something else here, maybe `;`